src/VersionTool/Program.cs

```diff
@@ -225,13 +253,324 @@ private static int OnList(CommandOption pathOption)
                 {
                     SaveJObject(project.ProjectFilePath, root);
 
-                    Console.WriteLine($"Updated: {project.ProjectFilePath}");
+                    //Console.WriteLine($"Updated: {project.ProjectFilePath}");
                 }
             }
 
             return 0;
         }
 
+        private static int OnUpdatePatch(
+            CommandOption directoryOption,
+            CommandOption udpatePatchConfigOption,
+            CommandOption updatePackageOption,
+            CommandOption updateRepoOption,
+            CommandArgument patchConfigArgument)
+        {
+            var directory = directoryOption.Value();
+            var patchConfigPath = patchConfigArgument.Value;
+            var patchConfig = LoadJObject(patchConfigPath);
```

Comment:
Let's deserialize this to a `PatchConfig` type. We know the structure, so no reason for it to loosely typed.